viewer/src/models/cad/determineSectors.ts

```diff
@@ -35,6 +35,7 @@ export interface DetermineSectorsByProximityInput {
 
 export function determineSectorsByProximity(params: DetermineSectorsByProximityInput): WantedSector[] {
   const hints = { ...defaultCadLoadingHints, ...(params.loadingHints || {}) };
+  console.log(hints.maxQuadSize);
```

Comment:
This seems like debugging output was pulled in together with the rest of the changes?

Reply:
oOoOOooOooooops 3.

I guess I passed Ballmers peak yesterday :|